src/main/java/org/padler/gradle/minify/MinifyPluginExtension.java

```diff
@@ -10,6 +10,10 @@
 
     private String jsSrcDir = "";
 
+    private Boolean createJsSourceMaps = false;
+    private Boolean createCssSourceMaps = false;
+    private Boolean originalFileNames = true;
```

Comment:
I guess default should be false because in most use cases it makes more sense to rename files to .min.*

Reply:
That was my plan but I somehow switched it accidentally.
Fixed now